nexus/src/app/disk.rs

```diff
@@ -408,4 +352,277 @@ impl super::Nexus {
 
         Ok(())
     }
+
+    /// Import blocks from a URL into a disk
+    pub async fn import_blocks_from_url_for_disk(
+        self: &Arc<Self>,
+        opctx: &OpContext,
+        disk_selector: &params::DiskSelector,
+        params: params::ImportBlocksFromUrl,
+    ) -> UpdateResult<()> {
+        let authz_disk: authz::Disk;
+
+        (_, _, _, authz_disk, _) = self
+            .disk_lookup(&opctx, &disk_selector)?
+            .fetch_for(authz::Action::Modify)
+            .await?;
```

Comment:
```suggestion
        (_, _, _, authz_disk) = self
            .disk_lookup(&opctx, &disk_selector)?
            .lookup_for(authz::Action::Modify)
            .await?;
```

Given you aren't actually using the db object, may as well only do the lookup.

Reply:
right, changed in 628ff59c